container-admin/Dockerfile_provisioner

```diff
@@ -55,6 +56,10 @@ RUN mkdir /consonance_logs && chmod a+rx /consonance_logs
 COPY bag_params.json /container-host-bag/example_params.json
 COPY example_tags.json /container-host-bag/example_tags.json
 
+# patch for Ansible
```

Comment:
Looks like the reason for the ansible code, this should explain why or better yet, link to a github issue.